tests/FSharp.Data.DesignTime.Tests/DocumentationTests.fs

```diff
@@ -71,10 +71,12 @@ for file in docFiles do
 
 [<Test>]
 [<TestCaseSource "docFiles">]
-[<Platform("Mono")>]
+//[<Platform("Mono")>]
```

Comment:
Is the platform attribute not supported anymore?

Reply:
I can't recall the problem I had with this but yes, I concluded it wasn't supported in .net standard .NUnit as yet.  I think it's an omission in the .Net standard NUnit DLL